CHANGELOG-1.3.md

```diff
@@ -0,0 +1,7 @@
+# Changelog since v1.2.0
+
+## Notable Changes
+
+- Snapshotter will no longer call ListSnapshots if the CSI Driver does not support this operation. ([#138](https://github.com/kubernetes-csi/external-snapshotter/pull/138), [@ggriffiths](https://github.com/ggriffiths))
+
+- Add a prefix "external-snapshotter-leader" and the driver name to the snapshotter leader election lock name. ([#129](https://github.com/kubernetes-csi/external-snapshotter/pull/129), [@zhucan](https://github.com/zhucan))
```

Comment:
Is this a potentially breaking change and wouldn't work with upgrades?

Reply:
It probably is.  The controller won't recognize old lock names any more.  Let me move this to a breaking change section.

Reply:
Can you also add some more description on what won't work (ie HA upgrades), any possible problems that could result (such as multiple snapshots created), and any possible mitigations?

Reply:
Updated.  PTAL.